Resources/Prototypes/Recipes/Construction/structures.yml

```diff
@@ -110,3 +110,62 @@
   objectType: Structure
   placementMode: SnapgridCenter
   canRotate: false
+
+- type: construction
+  name: Firelock
+  id: Firelock
+  graph: Firelock
+  startNode: start
+  targetNode: Firelock
+  category: Structures
+  description: This is a firelock - it locks an area when a fire alarm in the area is triggered. Don't get squished!
+  icon:
+    sprite: Constructible/Structures/Doors/firelock.rsi
+    state: closed
+  objectType: Structure
+  placementMode: SnapgridCenter
+
+- type: construction
+  name: Table Frame
+  id: TableFrame
+  graph: Tables
+  startNode: start
+  targetNode: TableFrame
+  category: Structures
+  description: A square piece of metal standing on four metal legs. (What did you expect?)
+  icon:
+    sprite: Constructible/Structures/Tables/frame.rsi
+    state: full
+  objectType: Structure
+  placementMode: SnapgridCenter
+
+- type: construction
+  name: Poker Table
+  id: PokerTable
+  graph: Tables
+  startNode: start
+  targetNode: PokerTable
+  category: Structures
+  description: A square piece of wood standing on four legs. (What did you expect?)
+  icon:
+    sprite: Constructible/Structures/Tables/carpet.rsi
+    state: full
+  objectType: Structure
+  placementMode: SnapgridCenter
+
+- type: construction
+  name: Catwalk
+  id: Catwalk
+  graph: Catwalk
+  startNode: start
+  targetNode: Catwalk
+  category: Structures
+  description: Just like a lattice except you can place wires on it.
+  conditions:
+    - !type:TileType
+      target: lattice
```

Comment:
You should be able to build this on top of tiles other than lattice. (See our maint: We use these and they look cool. Wires go under it.)